src/System.Reflection.Context/src/System.Reflection.Context.csproj

```diff
@@ -12,7 +12,6 @@
          but do not include this assembly in their framework.
```

Comment:
Ok, if not part of netstandard.dll then we'll need to continue to ship a package in order to expose it as portable.  I'll revert this guy for now.  We can decide later if we want to continue to ship that package.